daemon/cmd/daemon.go

```diff
@@ -824,14 +824,6 @@ func NewDaemon(ctx context.Context, cancel context.CancelFunc, epMgr *endpointma
 		return nil, nil, fmt.Errorf("failed to finalise LB initialization: %w", err)
 	}
 
-	if k8s.IsEnabled() {
-		bootstrapStats.k8sInit.Start()
-		// Launch the K8s watchers in parallel as we continue to process other
-		// daemon options.
-		d.k8sCachesSynced = d.k8sWatcher.InitK8sSubsystem(d.ctx)
-		bootstrapStats.k8sInit.End(true)
-	}
```

Comment:
Looks like the this hunk is added in commit `datapath: Runtime detection of devices` and then removed in the successive commit `daemon: Reload datapath on device changes` again. The same hunk is already present a few lines down. Is this addition/removal intentional?